django/core/management/commands/squashmigrations.py

```diff
@@ -36,6 +37,14 @@ def add_arguments(self, parser):
             "migration_name",
             help="Migrations will be squashed until and including this migration.",
         )
+        parser.add_argument(
+            "--ignore-dependencies",
+            "--ignore-deps",
+            action="store_false",
```

Comment:
```suggestion
            action="store_true",
```
(store_false is when the sense of the variable is reversed from the sense of the flag, e.g. below where the flag is called "no-input" but the variable is called "interactive")